cmd/otelcol/main.go

```diff
@@ -21,13 +21,13 @@ import (
 	"log"
 
 	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/internal/testcomponents"
 	"go.opentelemetry.io/collector/internal/version"
 	"go.opentelemetry.io/collector/service"
-	"go.opentelemetry.io/collector/service/defaultcomponents"
 )
 
 func main() {
-	factories, err := defaultcomponents.Components()
+	factories, err := testcomponents.DefaultComponents()
```

Comment:
Cannot depend on a "testing" package from the main file.

Reply:
Isn't this going away anyways?  Does it matter?